test/libsolidity/syntaxTests/array/uninitialized_storage_var.sol

```diff
@@ -0,0 +1,9 @@
+contract C {
+	function f() {
+		uint[] storage x;
+		uint[10] storage y;
```

Comment:
Could you please also add a mapping? We already noticed in a different place that e.g. `mapping(uint=>uint) x` will not trigger the "you have to add the storage keyword" warning, so maybe we also missed something here.

Reply:
It's already there https://github.com/ethereum/solidity/blob/9f59978c1d26d9ee691ea5d945bf703947cf38dc/test/libsolidity/syntaxTests/nameAndTypeResolution/211_uninitialized_mapping_array_variable.sol#L3

Reply:
Ah, no, that's an array of mappings, sorry.

Reply:
However, there's also https://github.com/ethereum/solidity/blob/9f59978c1d26d9ee691ea5d945bf703947cf38dc/test/libsolidity/syntaxTests/nameAndTypeResolution/210_uninitialized_mapping_variable.sol#L3 already - so I'm not sure what to add here...

Reply:
There's also 233_non_initialized_references.sol for structs already.

Reply:
ok!